### ISSUE TITLE
Commit the Status to Kafka only after the Scripts Succeeded in Computing the Data

### ISSUE THREAD

**Issue by VincenzoFerme** [benchflow/data-analyses-scheduler#44]
We now [commit the status to Kafka](https://github.com/benchflow/spark-tasks-sender/blob/dev/src/cloud/benchflow/sender/sender.go#L197) as soon as we start the computation from the message we receive. We should move this commit so that it happens only after the data are correctly handled by the scripts. We should do that by first evaluating the impact this change has on the current logic.

**Comment by VincenzoFerme** [comment]
Already done
